packages/frontend-shared/src/composables/useAddProject.ts

```diff
@@ -0,0 +1,37 @@
+import { gql } from '@urql/core'
+import { useMutation } from '@urql/vue'
+
+function makeAddProjectQuery (query?: string) {
+  const projectsQuery = `
+    projects {
+      id
+      title
+      projectId
+      projectRoot
+      isFirstTimeCT
+      isFirstTimeE2E
+    }
+  `
+
+  return gql`
+    mutation addProject($path: String!) {
```

Comment:
Assuming we're keeping this co-located with the component, we should be prefixing operations with the component name, e.g. `GlobalPageHeader_addProject`. Going to look to make some some changes to the network layer route so we get value out of this pattern when inspecting the network tab, e.g. `POST /graphql/GlobalPageHeader_addProject` rather than `POST /graphql`